src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+  color: #3d4e61;
+}
+
+.page {
+  margin: 0;
+  box-sizing: border-box;
+}
```

Comment:
Add here padding 8px for inline paddings

Reply:
got it